DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Indigo\Bundle\CookieConsentBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
+ */
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('indigo_cookie_consent');
+
+        $rootNode
+            ->children()
+                ->arrayNode('options')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->scalarNode('message')->defaultValue('message')->end()
+                        ->scalarNode('dismiss')->defaultValue('dismiss')->end()
+                        ->scalarNode('learnMore')->defaultValue('learn_more')->end()
+                        ->scalarNode('link')->end()
+                        ->scalarNode('container')->end()
+                        ->scalarNode('theme')->end()
```

Comment:
oh, and they go to the javascript lib as well? then i guess its ok like this - maybe add an info to each saying that this parameter is passed to the js lib and documented there?